public/index.html

```diff
@@ -35,11 +35,7 @@
         </script>
       </div>
 
-      <div id="root">
-        <p class="text-center text-large">
-          Please wait…
```

Comment:
The 'Please wait' message already comes from SendVerificationCode and SendVerificationEmail. I think it makes sense to wait to show it until you are actually waiting on something (like after main is called, when it's sending the code || email).

Reply:
Please make sure we have something on the screen while the browser downloads whatever we have on the page (script, mostly) on very slow connections. Not having any text on the page may seem that it's broken.

To test slow connections, you may use Network Link Conditioner, available via [Xcode Additional Tools](https://developer.apple.com/download/more/?q=Additional%20Tools).

Reply:
Ah, ok, makes sense. I'll add this back in!